examples/backtest/use_clock_timer/intro.md

```diff
@@ -0,0 +1,11 @@
+This is a simple example of how to use NautilusTrader's **Timer** feature in a strategy.
```

Comment:
Since we have dedicated directories for these examples, we could consider naming the file `README.md`? then when browsing on GitHub it will automatically render the markdown.

Reply:
Could it be worth considering prefixes like `001_use_clock_timer` to sort the examples in a more logical order than alpha?

Reply:
Done, but MYPY complained, after adding number at the beginning of the folder, so I had to fix it (make MYPY happy) with additional `examples_` prefix, as explained in later comment in this thread